### PR TITLE
Accelnet custom image

### PR DESCRIPTION
**Pull request by ameliashull-microsoft-gh**

### CODE REVIEW

File: articles/virtual-network/accelerated-networking-overview.md [hunk 1]
```diff
@@ -100,9 +100,18 @@ You can directly query the list of VM SKUs that support Accelerated Networking b
...
+If you use a custom image that supports Accelerated Networking, make sure you meet the following requirements.
 
-Images with cloud-init version 19.4 or later have networking correctly configured to support Accelerated Networking during provisioning.
+#### Device and driver support
+Any custom image supporting AccelNet must include drivers that enable Single Root I/O Virtualization for the network interface cards (NIC) which are used on Azure platforms. This hardware list includes NVIDIA [ConnectX-3](https://network.nvidia.com/pdf/user_manuals/ConnectX-3_Ethernet_Single_and_Dual_SFP+_Port_Adapter_Card_User_Manual.pdf), [ConnectX-4 Lx](https://docs.nvidia.com/networking/display/cx4lxen), [ConnectX-5](https://docs.nvidia.com/networking/display/connectx5en) and the [Microsoft Azure Network Adapter (MANA)](https://learn.microsoft.com/en-us/azure/virtual-network/accelerated-networking-mana-overview). Download and install the latest versions of these drivers into your custom image.
```
Comment (ameliashull-microsoft-gh) [inline articles/virtual-network/accelerated-networking-overview.md]: Remove NVIDIA docs

File: articles/virtual-network/accelerated-networking-overview.md [hunk 2]
```diff
@@ -153,6 +162,17 @@ EOF
 >[!NOTE]
```
Comment (ameliashull-microsoft-gh) [inline articles/virtual-network/accelerated-networking-overview.md]: Move note to just be sentence in paragraph before tab stuff

File: articles/virtual-network/accelerated-networking-overview.md [hunk 3]
```diff
@@ -153,6 +162,17 @@ EOF
...
 >We strongly advise that you don't run competing network interface software (such as ifupdown and networkd) on custom images, and that you don't run dhcpclient directly on multiple interfaces.
 
+#### Verify data path
+To ensure your custom image will support AccelNet, confirm that networking traffic is using the AccelNet data path.
```
Comment (ameliashull-microsoft-gh) [inline articles/virtual-network/accelerated-networking-overview.md]: Everywhere: spell out "Accelerated Networking"
Remove line

File: articles/virtual-network/accelerated-networking-overview.md [hunk 4]
```diff
@@ -153,6 +162,17 @@ EOF
 >[!NOTE]
 >We strongly advise that you don't run competing network interface software (such as ifupdown and networkd) on custom images, and that you don't run dhcpclient directly on multiple interfaces.
 
+#### Verify data path
```
Comment (ameliashull-microsoft-gh) [inline articles/virtual-network/accelerated-networking-overview.md]: Network traffic uses the Accelerated Networking data path

File: articles/virtual-network/create-vm-accelerated-networking-cli.md [hunk 5]
```diff
@@ -197,6 +197,11 @@ Once you create the VM in Azure, connect to the VM and confirm that the Ethernet
 
    The presence of the adapter confirms that Accelerated Networking is enabled for your VM.
 
+1. Verify the packets are flowing over the VF interface from the output of the following command:
+   ```powershell
+   PS C:\ > Get-NetAdapter | Where-Object InterfaceDescription –like "*Mellanox*Virtual*" | Get-NetAdapterStatistics
+   ```
```
Comment (ameliashull-microsoft-gh) [inline articles/virtual-network/create-vm-accelerated-networking-cli.md:207]: Add result of command also

File: articles/virtual-network/accelerated-networking-overview.md [hunk 6]
```diff
@@ -100,9 +100,18 @@ You can directly query the list of VM SKUs that support Accelerated Networking b
 
 ### Custom VM images
 
-If you use a custom image that supports Accelerated Networking, make sure you have the required drivers to work with Mellanox ConnectX-3, ConnectX-4 Lx, and ConnectX-5 NICs on Azure. Accelerated Networking also requires network configurations that exempt configuration of the virtual functions on the mlx4_en and mlx5_core drivers.
+If you use a custom image that supports Accelerated Networking, make sure you meet the following requirements.
 
-Images with cloud-init version 19.4 or later have networking correctly configured to support Accelerated Networking during provisioning.
+#### Device and driver support
+Any custom image supporting Accelerated Networking must include drivers that enable Single Root I/O Virtualization for the network interface cards (NIC) which are used on Azure platforms. This hardware list includes NVIDIA ConnectX-3, ConnectX-4 Lx, ConnectX-5 and the [Microsoft Azure Network Adapter (MANA)](https://learn.microsoft.com/en-us/azure/virtual-network/accelerated-networking-mana-overview). Download and install the latest versions of these drivers into your custom image.
```
Comment (mnanakul) [inline articles/virtual-network/accelerated-networking-overview.md]: I suppose we should just remove: 
Download and install the latest versions of these drivers into your custom image.

Comment (ameliashull-microsoft-gh) [reply articles/virtual-network/accelerated-networking-overview.md]: Removed it, is there anything we'd want to say instead?

**Pull request merged**